pages/signup.tsx

```diff
@@ -41,15 +46,15 @@ const ErrorMessage: React.FC<ErrorDisplayProps> = ({ signupErrors }) => {
   return <>{errorMessages}</>
 }
 
-const SignupSuccess: React.FC = () => (
+const SignupSuccess: React.FC<SignupSuccessProps> = ({ forgotToken }) => (
   <Card
     type="success"
     data-testid="signup-success"
     title="Account created successfully!"
   >
     <p>
-      You will receive a link to confirm your email address and to set your
-      password.
+      Link to set your password:
+      <a>{`https://www.c0d3.com/confirm/${forgotToken}`}</a>
```

Comment:
A button would be better, also use relative link.

```jsx
<NavLink path={`/confirm/${forgotToken}`} className="btn btn-primary">
  Click here to set your password.
</NavLink>
```

Reply:
it also appears I've forgotten how anchor tags work